operator/test/component/app/app_suite_test.go

```diff
@@ -32,7 +32,8 @@ var (
 )
 
 var _ = BeforeSuite(func() {
-	ctx, k8sManager, tracer, spanRecorder, k8sClient, _ = common.InitSuite()
+	var readyToStart chan struct{}
```

Comment:
maybe to improve readability
```suggestion
	var readyToStart chan bool
```

Reply:
I think it is less confusing to have an empty struct, with a bool I would expect to have something sent over, with an empty struct I would already know this is a sync channel

Reply:
agree with @RealAnna , as for signal-only channels empty struct channels are the preferred way in general